roles/wazuh/ansible-wazuh-manager/tasks/Debian.yml

```diff
@@ -28,6 +28,7 @@
 - name: Debian/Ubuntu | Installing Wazuh repository key
   apt_key:
     url: "{{ wazuh_manager_config.repo.gpg }}"
+    id: "{{ wazuhrepo.key_id }}"
```

Comment:
Please initialize the variable in the defaults installation for Manager and Agent.